docs/DeveloperGuide.md

```diff
@@ -34,7 +34,8 @@ title: Developer Guide
     - [3.3. List Features](#33-list-features)
         * [3.3.1 Implementation](#331-implementation)
         * [3.3.2 Design Consideration - **List Recipes**](#332-design-consideration-list-recipe)
-            * [Aspect: Concern while adding a new feature](#3321-aspect)
+            * [Aspect 1 : Concern while adding a new feature](#3321-aspect-1)
+            * [Aspect 2: What and how to show recipe information](#3322-aspect-2)
```

Comment:
what do you mean what and how?

Reply:
lol what fields of recipe to display and how to display some fields that could be very long

Reply:
ooh maybe can word beter? like what info to show and how to show them?

Reply:
okok